shared/src/StatusCodes.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Devices.Shared
+{
+    /// <summary>
+    /// A list of common status codes to represent the response from the client.
+    /// </summary>
+    /// <remarks>
+    /// These status codes are based on the HTTP status codes listed here <see href="http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml"/>
+    /// </remarks>
+    public static class StatusCodes
```

Comment:
can you extend a static class? If I saw this class I'd want to extend it to add my custom codes in here.

Reply:
I can't think of a different approach that I prefer, even a partial class isn't the right option since it needs to be in the same assembly and namespace